apple-nav/src/components/Displays/IndividualProducts/NewIpad.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+
+
+function NewIpad(props) {
+  
+  const item = props.ipad.find(each => each.name === props.match.params.apple);
```

Comment:
This is a really neat idea. I like it. And shows you understand really how params work.